CONTRIBUTING.rst

```diff
@@ -190,6 +190,11 @@ Running System Tests
    $ export CLOUDSDK_PYTHON_SITEPACKAGES=1
 
    # Authenticate the gcloud tool with your account.
+   $ P12_CREDENTIALS_FILE="path/to/app_credentials.json"
```

Comment:
Wouldn't it be less confusing to use a name like `JSON_CREDENTIALS_FILE` for the non-P12 example?